src/helpers/handlers.ts

```diff
@@ -111,7 +110,7 @@ export const queryResultStream = (response: ReadableStream<ArrayBuffer>) => {
             }
           }
         }
-        dataBuffer = lines[lines.length - 1]
+        dataBuffer = lines.length > 1 ? lines[lines.length - 1] : ''
```

Comment:
This make sure we 'flush' extra json lines if we on'y had 1 to begin with. An enhancement here (which I've created a ticket for) would be to actually 'pop' each line out of the 'buffer'.